clang/include/clang/Basic/AttrDocs.td

```diff
@@ -2212,6 +2212,40 @@ device kernel, the attribute is ignored and it is not propagated to a kernel.
   }];
 }
 
+def SYCLIntelStallEnableAttrDocs : Documentation {
+  let Category = DocCatFunction;
+  let Heading = "intel::stall_enable";
+  let Content = [{
+When applied to a lambda or function call operator (of a function object)
+on device, this requests, to the extent possible, that statically-scheduled
+clusters handle stalls using a stall-enable signal to freeze computation
+within the cluster.
+
+If ``intel::stall_enable`` is applied to a function called from a device
+kernel, the attribute is ignored and it is not propagated to a kernel.
+
+The ``intel::stall_enable`` attribute has an effect when applied to a
+function, and no effect otherwise. This attribute takes no argument and
+LLVM IR function metadata should simply be i32 1.
```

Comment:
Do we really need to mention what will be emitted in LLVM IR?

Reply:
I do not think we need to mention this. Removed it.